x-pack/plugins/security_solution/public/timelines/components/timeline/selectable_timeline/index.tsx

```diff
@@ -210,7 +212,7 @@ const SelectableTimelineComponent: React.FC<SelectableTimelineProps> = ({
   const searchProps: EuiSelectableProps['searchProps'] = useMemo(
     () => ({
       'data-test-subj': 'timeline-super-select-search-box',
-      placeholder: i18n.SEARCH_BOX_TIMELINE_PLACEHOLDER(timelineType),
+      placeholder: placeholder ?? i18n.SEARCH_BOX_TIMELINE_PLACEHOLDER(timelineType),
       onSearch: onSearchTimeline,
       incremental: true,
```

Comment:
```suggestion
      incremental: false,
```

So in testing I found this other bug where the partial search suggestions would disappear after a second or two, and then suggestions wouldn't show up again until there was full word match: 
<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/2946766/178598977-26ac02a7-c9a6-45f5-bae5-3988c591ffc5.gif" />
</p>


Whatever the case, switching to `incremental: false` seems to resolve this issue 🎉 
<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/2946766/178599509-6f1526d0-2d11-48eb-884c-ca2deaa85005.gif" />
</p>